packages/postcss/mixin.core.js

```diff
@@ -93,14 +96,17 @@ class PostCSSMixin extends Mixin {
     allLoaderConfigs.splice(
       allLoaderConfigs.indexOf(jsLoaderConfig),
       0,
-      getCSSLoaderConfig(this.config.browsers, ExtractCSSPlugin.loader)
-    );
-
-    webpackConfig.plugins.unshift(
-      new FilterWarningsPlugin({
-        exclude: /\[mini-css-extract-plugin\][^]*Conflicting order between:/,
+      getCSSLoaderConfig(this.config.browsers, {
+        loader: ExtractCSSPlugin.loader,
```

Comment:
`mini-css-extract-plugin`'s loader apparently needs a little configuration now to work as expected.